Dev-Typescript (#1322)
twilio/twilio-video.js: README.md

const Video = require('twilio-video');
```

> TypeScript definitions can now be imported using this method.

Reviewer: ```suggestion
TypeScript definitions can now be imported as shown below.
```
